index.html

```diff
@@ -64,6 +64,10 @@
 				<h3>Open Source</h3>
 				<p>The Lounge is open source and licensed under the MIT license.</p>
 			</div>
+			<div class="col-sm-4 feature">
+				<h3>Community Managed</h3>
+				<p>The Lounge is an active community fork of <a href="http://www.shout-irc.com">Shout</a>. It's development is made possible by a growing and dedicated community. You can find us in #thelounge on Freenode.</p>
```

Comment:
Actually, this is too big compared to the other blocks, I think:

<img width="751" alt="screen shot 2016-09-19 at 20 37 50" src="https://cloud.githubusercontent.com/assets/113730/18653606/ff4f1514-7ea8-11e6-981c-ba9e6a564d33.png">

What about:

```html
It is run by a growing and dedicated community as a fork of <a href="http://www.shout-irc.com">Shout</a>. Find us in <code>#thelounge</code> on Freenode.</p>
```

Which gives:

<img width="759" alt="screen shot 2016-09-19 at 20 50 00" src="https://cloud.githubusercontent.com/assets/113730/18653835/aa896f0a-7eaa-11e6-8962-8f35cab584be.png">

Not perfect but I don't have anything better...